README.md

```diff
@@ -1015,113 +1103,157 @@ Constructs equi-width histograms, in which the histogram range is divided into i
 
 - `min_value` and `max_value`: The low and high end points of the acceptable range for the expression. The end points must also evaluate to numeric values and not be equal.
 
 - `num_buckets`:  The desired number of buckets; must be a positive integer value. A value from the expression is assigned to each bucket, and the function then returns the corresponding bucket number.
 
 When an expression falls outside the range, the function returns:
+
 - `0` if the expression is less than min_value.
 - `num_buckets + 1` if the expression is greater than or equal to max_value.
 
-
 **Usage:**
+
 ```
 {{ dbt_utils.width_bucket(expr, min_value, max_value, num_buckets) }}
 ```
 
 ### Web macros
+
 #### get_url_parameter ([source](macros/web/get_url_parameter.sql))
+
 This macro extracts a url parameter from a column containing a url.
 
 **Usage:**
+
 ```
 {{ dbt_utils.get_url_parameter(field='page_url', url_parameter='utm_source') }}
 ```
 
 #### get_url_host ([source](macros/web/get_url_host.sql))
+
 This macro extracts a hostname from a column containing a url.
 
 **Usage:**
+
 ```
 {{ dbt_utils.get_url_host(field='page_url') }}
 ```
 
 #### get_url_path ([source](macros/web/get_url_path.sql))
+
 This macro extracts a page path from a column containing a url.
 
 **Usage:**
+
 ```
 {{ dbt_utils.get_url_path(field='page_url') }}
 ```
+
 ----
+
 ### Cross-database macros
+
 These macros make it easier for package authors (especially those writing modeling packages) to implement cross-database
 compatibility. In general, you should not use these macros in your own dbt project (unless it is a package)
 
+Note that most of these macros have now moved to dbt-core. Instead of the `dbt_utils.` namespace, you should now call these macros with the `dbt.` prefix, or without a prefix altogether (as shown in the examples below). They will eventuall be removed from `dbt_utils`.
+As highlighted below, some of the cross-database macros are still in the process of being deprecated.
+
 #### current_timestamp ([source](macros/cross_db_utils/current_timestamp.sql))
+
+*DEPRECATED: This macro is deprecated and will be removed in a future version of the package, once equivalent functionality is implemented in dbt Core.*
+
 This macro returns the current timestamp.
 
 **Usage:**
+
 ```
 {{ dbt_utils.current_timestamp() }}
 ```
 
 #### dateadd ([source](macros/cross_db_utils/dateadd.sql))
+
+*DEPRECATED: This macro is now provided in dbt Core. It is no longer available in dbt_utils and backwards compatibility will be removed in a future version of the package.*
+
 This macro adds a time/day interval to the supplied date/timestamp. Note: The `datepart` argument is database-specific.
 
 **Usage:**
+
 ```
-{{ dbt_utils.dateadd(datepart='day', interval=1, from_date_or_timestamp="'2017-01-01'") }}
+{{ dateadd(datepart='day', interval=1, from_date_or_timestamp="'2017-01-01'") }}
```

Comment:
```suggestion
{{ dbt_utils.dateadd(datepart='day', interval=1, from_date_or_timestamp="'2017-01-01'") }}
```

As discussed above - since these are the docs for dbt_utils, I think we continue to show the syntax that _works_ in dbt utils (but I think the deprecation warning is great), especially since people generally access the readme on `main` regardless of which version of utils they actually have installed. 

(This feedback applies to all the other macros too, but I don't think there's any use in copy-pasting it across every single one)

Reply:
Makes sense, I'll update the rest 👍